File: hamersapp/src/main/java/nl/ecci/hamers/beers/Beer.java
package nl.ecci.hamers.beers;

import com.google.gson.annotations.SerializedName;

import java.util.Date;

public class Beer {

    public static final String BEER = "BEER";

    private final int id;
    private final String name;
    @SerializedName("soort")
    private final String kind;
    @SerializedName("picture")
    private final String imageURL;
    private final String percentage;
    private final String brewer;
    private final String country;
    @SerializedName("cijfer")
    private final String rating;
    private final String url;
    @SerializedName("created_at")
    private final Date createdAt;

    @SuppressWarnings("SameParameterValue")
    public Beer(int id, String name, String kind, String imageURL, String percentage, String brewer, String country, String rating, String url, Date createdAt) {
        super();
        this.id = id;
        this.name = name;
        this.kind = kind;
        this.imageURL = imageURL;
        this.percentage = percentage;
        this.brewer = brewer;
        this.country = country;
        this.rating = rating;
        this.url = url;
        this.createdAt = createdAt;
    }

    public int getID() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getKind() {
        return kind;
    }

    public String getImageURL() {
        return imageURL;
    }

    public String getPercentage() {
        return percentage;
    }

    public String getBrewer() {
        return brewer;
    }

    public String getCountry() {
        return country;
    }

    public String getRating() {
        return rating;
    }

    public String getUrl() {
        return url;
    }

    public Date getCreatedAt() {
        return createdAt;
    }
}

File: hamersapp/src/main/java/nl/ecci/hamers/beers/Beer.kt
package nl.ecci.hamers.beers

import com.google.gson.annotations.SerializedName
import nl.ecci.hamers.helpers.Utils
import java.util.*

class Beer {
    val id: Int = Utils.notFound
    var name: String = Utils.unknown
    @SerializedName("soort")
    var kind: String = Utils.unknown
    @SerializedName("picture")
    val imageURL: String = Utils.unknown
    var percentage: String = Utils.unknown
    var brewer: String = Utils.unknown
    var country: String = Utils.unknown
    @SerializedName("cijfer")
    var rating: String = Utils.unknown
    val url: String = Utils.unknown
    @SerializedName("created_at")
    val createdAt: Date = Date()

    companion object {
        val BEER = "BEER"
    }
}